project/Smithy4sBuildPlugin.scala

```diff
@@ -390,6 +391,20 @@ object Smithy4sBuildPlugin extends AutoPlugin {
   }
 
   lazy val jsDimSettings = simpleJSLayout ++ Seq(
+    scalacOptions ++= {
+      // Map the sourcemaps to github paths instead of local directories
+      val flag =
+        if (scalaVersion.value.startsWith("3")) "-scalajs-mapSourceURI"
+        else "-P:scalajs:mapSourceURI"
+      val localSourcesPath = baseDirectory.value.toURI
```

Comment:
This should be `(LocalRootProject / baseDirectory).value.toURI.toString`, exactly like it is in sbt-typelevel.

https://github.com/typelevel/sbt-typelevel/blob/4da726ca021ea2c6459e8286d9ec49ab7c79a883/github/src/main/scala/org/typelevel/sbt/TypelevelScalaJSGitHubPlugin.scala#L40

Reply:
You're my hero.
I'll try to get a PR that fixes it today!

Reply:
thanks @armanbilge <#